### ISSUE TITLE
Duplicate timestamp in ubuntu syslog

### ISSUE THREAD

**Issue by VMelnalksnis** [VMelnalksnis/Gnomeshade#361]
**Describe the bug**
The timestamp of the event is logged twice for console sink on Ubuntu.

**To Reproduce**
1. Run WebApi on Ubuntu
2. Run `sudo journalctl -u gnomeshade -f`
3. See logs with the same timestamp twice before the message
```
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] Start processing HTTP request GET https://ob.nordigen.com/api/v2/institutions/SANDBOXFINANCE_SFIN0000/
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] Sending HTTP request GET https://ob.nordigen.com/api/v2/institutions/SANDBOXFINANCE_SFIN0000/
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] Received HTTP response headers after 1.9439ms - 200
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] End processing HTTP request after 2.0371ms - 200
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] Received HTTP response headers after 85.4793ms - 200
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] End processing HTTP request after 86.3498ms - 200
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] Executed endpoint 'Health checks'
Jul 18 11:09:30 gnomeshade Gnomeshade.Interfaces.WebApi[13940]: [11:09:30 INF] HTTP GET /health responded 200 in 87.6851 ms
```

**Expected behavior**
The timestamp is logged only once.

**Comment by VMelnalksnis** [comment]
Moved console logging config from code to configuration. The message can now be set (example with default template):
```
{
  "Name": "Console"
  "Args": {
    "outputTemplate": "[{Timestamp:HH:mm:ss} {Level:u3}] {Message:lj}{NewLine}{Exception}"
  }
}
```

**Comment by VMelnalksnis** [comment]
This worked locally on Windows, but now there is no output on ubuntu.

**Comment by VMelnalksnis** [comment]
Fixed by #727 by moving away from Serilog.
